libgcrypt-i686-w64-mingw32.static libgcrypt-1.9.2 build error under WSL

/mxe/tmp-libgpg_error-i686-w64-mingw32.static/libgpg-error-1.42.build_/src/../../libgpg-error-1.42/src/logging.c:458: undefined reference to `_imp__closesocket@4'

/mxe/usr/lib/gcc/i686-w64-mingw32.static/5.5.0/../../../../i686-w64-mingw32.static/lib/../lib/libgpg-error.a(libgpg_error_la-logging.o): In function `writen':

/mxe/tmp-libgpg_error-i686-w64-mingw32.static/libgpg-error-1.42.build_/src/../../libgpg-error-1.42/src/logging.c:169: undefined reference to `_imp__send@16'

/mxe/usr/lib/gcc/i686-w64-mingw32.static/5.5.0/../../../../i686-w64-mingw32.static/lib/../lib/libgpg-error.a(libgpg_error_la-logging.o): In function `fun_writer':

/mxe/tmp-libgpg_error-i686-w64-mingw32.static/libgpg-error-1.42.build_/src/../../libgpg-error-1.42/src/logging.c:443: undefined reference to `_imp__closesocket@4'

/mxe/tmp-libgpg_error-i686-w64-mingw32.static/libgpg-error-1.42.build_/src/../../libgpg-error-1.42/src/logging.c:342: undefined reference to `_imp__htons@4'

/mxe/tmp-libgpg_error-i686-w64-mingw32.static/libgpg-error-1.42.build_/src/../../libgpg-error-1.42/src/logging.c:361: undefined reference to `_imp__inet_addr@4'

/mxe/tmp-libgpg_error-i686-w64-mingw32.static/libgpg-error-1.42.build_/src/../../libgpg-error-1.42/src/logging.c:370: undefined reference to `_imp__socket@12'

/mxe/tmp-libgpg_error-i686-w64-mingw32.static/libgpg-error-1.42.build_/src/../../libgpg-error-1.42/src/logging.c:381: undefined reference to `_imp__connect@12'

/mxe/tmp-libgpg_error-i686-w64-mingw32.static/libgpg-error-1.42.build_/src/../../libgpg-error-1.42/src/logging.c:387: undefined reference to `_imp__closesocket@4'

--- a/src/libgcrypt.mk
+++ b/src/libgcrypt.mk
@@ -3,8 +3,8 @@
 PKG             := libgcrypt
 $(PKG)_WEBSITE  := https://directory.fsf.org/wiki/Libgcrypt
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 1.9.2
-$(PKG)_CHECKSUM := b2c10d091513b271e47177274607b1ffba3d95b188bbfa8797f948aec9053c5a
+$(PKG)_VERSION  := 1.9.3
+$(PKG)_CHECKSUM := 97ebe4f94e2f7e35b752194ce15a0f3c66324e0ff6af26659bbfb5ff2ec328fd
 $(PKG)_SUBDIR   := libgcrypt-$($(PKG)_VERSION)
 $(PKG)_FILE     := libgcrypt-$($(PKG)_VERSION).tar.bz2
 $(PKG)_URL      := https://gnupg.org/ftp/gcrypt/libgcrypt/$($(PKG)_FILE)
@@ -34,7 +34,9 @@ define $(PKG)_MAKE
      echo 'Version: $($(PKG)_VERSION)'; \
      echo 'Description: $(PKG)'; \
      echo 'Libs: ' "`$(TARGET)-libgcrypt-config --libs`"; \
-     echo 'Cflags: ' "`$(TARGET)-libgcrypt-config --cflags`";) \
+     echo 'Cflags: ' "`$(TARGET)-libgcrypt-config --cflags`"; \
+     echo 'Libs: -lws2_32'; \
+     ) \
      > '$(PREFIX)/$(TARGET)/lib/pkgconfig/$(PKG).pc'
 
     '$(TARGET)-gcc' \

